hlf/chaincode/admission.md

```diff
@@ -0,0 +1,165 @@
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## ContractName "UC4.Admission"
+
+## Transactions
+
+### Add Admission
+- ID = addAdmission
+- Send
+    - enrollmentId :: String
+    - courseId :: String
+    - moduleId :: String
+    - timestamp :: String \<DATE ISO 8601 YYYY-MM-DDThh:mm:ss\>
+- Receive
+    - [Admission](#Admission)
+      -  Description: Done, returns the submitted data, decorated with an admissionId.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes within the json are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [parameter checks](#parameterChecks).
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLInvalidEntity",
+        "title": "The following parameters produced some semantic error",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+      - Description: This error is returned, if the given parameters produce some error.  
+      For detailed informations see [semantic checks](#semanticChecks).
+
+
+### Drop Admission
+- ID = dropAdmission
+- Send
+    - admissionId :: String
+- Receive
+    - ""
+      -  Description: Done, returns the submitted data.
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLNotFound",
+        "title": "There is no Admission for the given admissionId"
+      }
+      ```
+      - Description: This error is returned, if the admissionId is not present on the ledger.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+      - Description: This error is returned, if the given parameters are not well formatted, they are listed in invalidParams.  
+      For detailed informations see [parameter checks](#parameterChecks).
+      (admissionId)
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLInsufficientApprovals",
+        "title": "The approvals present on the ledger do not suffice to execute this transaction"
+      }
+      ```
+      - Description: This error is returned, if the required approvals are not present.

```

Comment:
Can this be returned from ```addAdmission``` as well?

Reply:
Yes, this should be added

Reply:
resolved in https://github.com/upb-uc4/api/pull/83/commits/e5665962dba5f6eac477bd3d20c0f18910f14137